vectornav/src/vn_sensor_msgs.cc

```diff
@@ -221,10 +231,10 @@ class VnSensorMsgs : public rclcpp::Node
       msg.header = msg_in->header;
 
       if(use_enu) {
-        convert_to_enu(msg_in, msg);
+        convert_to_enu(msg_in, msg, false);
```

Comment:
I'm not sure about changing the default behavior here...

It might be ok if this were some sort of parameter to use uncompensated data, but I feel that the default should be to use compensated data.

What are your thoughts?

Reply:
And I set the third parameter of use_enu to true by default to use compensated measurements by default